server/templates/server/template.html

```diff
@@ -57,9 +57,11 @@
                 </li>
             </ul>
             <ul class="navbar-nav mr-auto left">
+                {% if user and user.is_authenticated %}
                 <li class="nav-item">
                     <a class="nav-link" href="{% url 'logout' %}">{{ user.username }}: התנתק</a>
                 </li>
```

Comment:
```suggestion
                    <li class="nav-item">
                        <a class="nav-link" href="{% url 'logout' %}">{{ user.username }}: התנתק</a>
                    </li>
```

Reply:
Personally, I don't add indentation between django template blocks because these blocks are removed after preprocessing, which would theoretically lead to a generated file with the <li> block in this example over-indented. Is this true? I don't know, but we never do this in {% block %} tags so why in ifs?
TBH I don't mind that much. If you really want this to be indented more I'll accept it.